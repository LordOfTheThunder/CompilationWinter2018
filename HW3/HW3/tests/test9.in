void TestShadowing() {
	int x = 3;
	{
		x = 5;
	}
}

void main() { print("Max is cool"); }
