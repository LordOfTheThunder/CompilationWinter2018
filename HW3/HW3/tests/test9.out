---end scope---
---end scope---
x INT 0
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
TestShadowing ()->VOID 0
